worldengine/common.py

```diff
@@ -120,3 +120,33 @@ def array_to_matrix(array, width, height):
         for x in range(width):
             matrix[y].append(array[y * width + x])
     return matrix
+
+def _equal(a, b):
```

Comment:
Is this function used in other parts of WE or just tests? If just tests, can it be moved to the tests please?